### PR TITLE
Add AWS course

### PR DESCRIPTION
**Pull request by fabianschmauder**

### CODE REVIEW

File: src/assets/samples/courses.json [hunk 1]
```diff
@@ -430,99 +430,107 @@
...
+          "Kubernetes"
         ]
       },
       {
-        "title": "Grundlagen des Webs",
+        "title": "Networking Concepts",
+        "items": ["IPv4 & IPV6", "Amazon VPC", "Amazon Route 53"]
+      },
+      {
+        "title": "Best practice",
```
Comment (macfrei) [inline src/assets/samples/courses.json:465]:
```suggestion
        "title": "Best Practice",
```

File: src/assets/samples/courses.json [hunk 2]
```diff
@@ -430,99 +430,107 @@
...
+          "Python Fundamentals ",
+          "Automation ",
+          "Continuous Integration (CI) mit GitHub Actions",
+          "Continuous Deployment (CD) mit GitHub Actions",
+          "JSON / YAML  ",
+          "VSCode  ",
+          "API und REST ",
+          "Git Workflow ",
+          "Markdown ",
+          "Pair Programming"
```
Comment (macfrei) [inline src/assets/samples/courses.json:533]: Don't know if that was on purpose or not 🤷‍♀️ 
```suggestion
          "Python Fundamentals",
          "Automation",
          "Continuous Integration (CI) mit GitHub Actions",
          "Continuous Deployment (CD) mit GitHub Actions",
          "JSON / YAML",
          "VSCode",
          "API und REST",
          "Git Workflow",
          "Markdown",
          "Pair Programming"
```

Comment (fabianschmauder) [reply src/assets/samples/courses.json]: thanks :)

**Review by macfrei — APPROVED** [review]

**Comment by jamarob** [comment]
Have you verified that this generates a certificate? As is the event page is still needed to generate the certificate 🤔

**Pull request merged**